apps/Unciv/uninstall

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+rm -f ~/.local/share/applications/unciv.desktop
+rm -rf ~/Unciv
+purge_packages || exit 1
```

Comment:
@Botspot fixed the mistakes that you pointed out. However, I am wondering what this line does. I copied it from another app entry that I was using as a reference. But don't know what it does.